cmake/Modules/FindRTIConnextDDS.cmake

```diff
@@ -1063,6 +1070,20 @@ function(get_all_library_variables
     )
 endfunction()
 
+# This macro adds the location property to a list of properties in order to
+# clean the ``create_connext_imported_target`` function.
+# Arguments:
+# - _list_var: List variable name to which append the location properties.
```

Comment:
I don't know how important the comments are (do users see them or just RTI people)? 

Anyway, do you mean, "List variable name to append to the location properties" or "List variable name to which to append the location properties" (notice the extra "to" before "append")? Those mean two different things. You probably mean the second one?

Reply:
What about :
> Name of the variable in which the location properties will be appended

Reply:
Use: 
List variable name to which to append the location properties.
OR:
List variable name in which to list the location properties.

Append means "add to the end of," so saying that we will append properties "in" a variable is not right. We never append "in," always "to."

Reply:
Alright! Thank you!